include/GafferUI/StandardStyle.h

```diff
@@ -98,6 +98,8 @@ class GAFFERUI_API StandardStyle : public Style
 		void renderRotateHandle( Axes axes, State state = NormalState ) const override;
 		void renderScaleHandle( Axes axes, State state = NormalState ) const override;
 
+    static void reloadShaders();
```

Comment:
Indentation. I know it's fixed in a later commit, but it still makes review harder...